src/measure_theory/function/conditional_expectation/basic.lean

```diff
@@ -2070,6 +2080,33 @@ begin
   rw [hg_eq s hs hμs, set_integral_condexp hm hf hs],
 end
 
+lemma condexp_bot' [is_finite_measure μ] (f : α → F') :
```

Comment:
This should still hold if $\mu$ is not finite right? Both sides are 0 since $\mu$ restricted on the $\perp$ will not be sigma finite so the LHS is 0.